app.go

```diff
@@ -209,8 +210,8 @@ func (app *App) OgpPage(w http.ResponseWriter, r *http.Request) {
 	return
 }
 
-// CreateImage create ogp image
-func (app *App) CreateImage(w http.ResponseWriter, r *http.Request) {
+// CreateImagePage create ogp image
```

Comment:
I assume this function creates a web page, not an image, and the function comment should follow the actual side effect that the function causes.